### ISSUE TITLE
Why the baseline performance is different with the original version? 

### ISSUE THREAD

**Issue by liu-jc** [oj9040/GraphSAGE_RL#1]
In the original version of [GraphSAGE](https://github.com/williamleif/GraphSAGE/tree/master/graphsage), the micro f1 score after 50 epochs in PPI dataset is only around 0.57. I don't modify the args in the shell script. In your experiments using uniform sampling, I use the exactly same args and the performance can achieve over 0.65. But why? I think they should be similar. Have you modified evaluation settings or done any improvement for uniform sampling?

**Comment by oj9040** [comment]
I double-checked the result by rerunning, but it still can achieve the accuracy in baseline as below.
Please confirm your computing environment for me.

 $ python -m graphsage.supervised_train --train_prefix ~/data/ppi/ppi --model mean_concat --dim_1 512 --dim_2 512 --samples_1 30 --samples_2 30 --batch_size 512 --validate_batch_size 512 --gpu 0 --sigmoid --epochs 10

 ** Uniform sampling (baseline) **
  205 Full validation stats: loss= 0.40309 f1_micro= 0.65616 time= 0.16346
  206 Full validation stats: loss= 0.40521 f1_micro= 0.65540 time= 0.17425
  207 Full validation stats: loss= 0.40573 f1_micro= 0.65120 time= 0.17537
  208 Full validation stats: loss= 0.40566 f1_micro= 0.65463 time= 0.17355
  209 Full validation stats: loss= 0.40445 f1_micro= 0.65604 time= 0.17142
  210 Full validation stats: loss= 0.40578 f1_micro= 0.65553 time= 0.19164
  211 Full validation stats: loss= 0.40494 f1_micro= 0.65608 time= 0.17293
  212 Full validation stats: loss= 0.40367 f1_micro= 0.65586 time= 0.17168
  213 Full validation stats: loss= 0.40681 f1_micro= 0.65232 time= 0.17346
  214 Full validation stats: loss= 0.40461 f1_micro= 0.65522 time= 0.18150
  215 mean: loss=0.40500 f1_micro=0.65484 time=0.17493

**Comment by oj9040** [comment]
In original GraphSAGE, the default hidden dimension is 128 if you did not specified additionally. However, my shell script specified to use 512. Maybe this caused the difference.
